clients/src/main/java/org/apache/kafka/common/requests/PushTelemetryRequest.java

```diff
@@ -83,8 +94,38 @@ public Uuid getClientInstanceId() {
         return this.data.clientInstanceId();
     }
 
+    public int getSubscriptionId() {
+        return this.data.subscriptionId();
+    }
+
+    public boolean isClientTerminating() {
+        return this.data.terminating();
+    }
+
+    public String getMetricsContentType() {
+        // Future versions of PushTelemetryRequest and GetTelemetrySubscriptionsRequest may include a content-type
+        // field to allow for updated OTLP format versions (or additional formats), but this field is currently not
+        // included since only one format is specified in the current proposal of the kip-714
+        return null;
```

Comment:
The Payload class's contentType() method says that it will report a proper string and not null, but maybe that's handled somewhere else?